routes/search.js

```diff
@@ -155,30 +155,18 @@ function sendCourseVersion(query, version, res) {
     .catch((err) => errorHandler(res, 400, err));
 }
 
-router.get("/api/getYearRange", async (req, res) => {
-  let years = { min: null, max: null };
-
-  let smallest = 2016;
-  let largest = new Date().getFullYear();
-
-  const terms = ["Fall", "Intersession", "Spring", "Summer"];
-
-  for (let i = smallest; i <= largest; i++) {
-    for (const term of terms) {
-      const resp = await SISCV.findOne({
-        "versions.term": { $in: [term + " " + i] },
-      });
-      if (resp) {
-        if (!years.min) {
-          years.min = i;
-        } else if (!years.max || years.max < i) {
-          years.max = i;
-        }
-      }
-    }
-  }
-
-  returnData(years, res);
+router.get("/api/getYearRange", (req, res) => {
```

Comment:
implementation is clean, would be a good practice to write some comments on what this route/the for each loop is doing